diagnostics/physics/sst_eval.py

```diff
@@ -70,55 +75,70 @@ def plot_sst_eval(pp_root):
+    logger.info("Successfully plotted model data")
 
     # OISST
-    grid[1].pcolormesh(oisst_lonc, oisst_latc, oisst_ave, cmap=cmap, norm=norm)
+    grid[1].pcolormesh(oisst_lonc, oisst_latc, oisst_ave, cmap=cmap, norm=norm, transform=proj)
     grid[1].set_title('(b) OISST')
+    logger.info("Successfully plotted oisst")
 
     # Model - OISST
-    grid[2].pcolormesh(oisst_lonc, oisst_latc, delta_oisst, **bias_common)
+    grid[2].pcolormesh(oisst_lonc, oisst_latc, delta_oisst, transform=proj,**bias_common)
     grid[2].set_title('(c) Model - OISST')
-    annotate_skill(mom_rg, oisst_ave, grid[2], dim=['lat', 'lon'])
+    annotate_skill(mom_rg, oisst_ave, grid[2], dim=['lat', 'lon'], x0=config['text_x'], y0=config['text_y'], yint=config['text_yint'])
+    logger.info("Successfully plotted difference between model and oisst")
 
     # GLORYS
-    p1 = grid[4].pcolormesh(glorys_lonc, glorys_latc, glorys_ave, cmap=cmap, norm=norm)
+    p1 = grid[4].pcolormesh(glorys_lonc, glorys_latc, glorys_ave, cmap=cmap, norm=norm, transform=proj)
     grid[4].set_title('(d) GLORYS12')
     cbar1 = autoextend_colorbar(grid.cbar_axes[1], p1)
     cbar1.ax.set_xlabel('Mean SST (°C)')
+    logger.info("Successfully plotted glorys")
 
     # Model - GLORYS
-    p2 = grid[5].pcolormesh(model_grid.geolon_c, model_grid.geolat_c, delta_glorys, **bias_common)
+    p2 = grid[5].pcolormesh(model_grid.geolon_c, model_grid.geolat_c, delta_glorys, transform=proj, **bias_common)
     cbar2 = autoextend_colorbar(grid.cbar_axes[2], p2)
     cbar2.ax.set_xlabel('SST difference (°C)')
     cbar2.ax.set_xticks([-2, -1, 0, 1, 2])
     grid[5].set_title('(e) Model - GLORYS12')
-    annotate_skill(model_ave, glorys_rg, grid[5], weights=model_grid.areacello)
+    annotate_skill(model_ave, glorys_rg, grid[5], weights=model_grid.areacello, x0=config['text_x'], y0=config['text_y'], yint=config['text_yint'])
+    logger.info("Successfully plotted difference between glorys and model")
 
     for ax in grid:
-        ax.set_xlim(-99, -35)
-        ax.set_ylim(4, 59)
+        ax.set_extent([ config['x']['min'], config['x']['max'], config['y']['min'], config['y']['max'] ], crs=proj )
         ax.set_xticks([])
         ax.set_yticks([])
         ax.set_xticklabels([])
         ax.set_yticklabels([])
         for s in ax.spines.values():
             s.set_visible(False)
-        
-    plt.savefig('figures/sst_eval.png', dpi=300, bbox_inches='tight')
+    logger.info("Successfully set extent of each axis")
+
+    plt.savefig(config['figures_dir']+'sst_eval.png', dpi=300, bbox_inches='tight')
+    logger.info("Successfully saved figure")
 
 
 if __name__ == '__main__':
     from argparse import ArgumentParser
     parser = ArgumentParser()
-    parser.add_argument('pp_root', help='Path to postprocessed data (up to but not including /pp/)')
+    parser.add_argument('-p','--pp_root', type=str, help='Path to postprocessed data (up to but not including /pp/)')
```

Comment:
Since these two command line arguments are flags but are not optional, you could add `required=True` to the `add_argument` call so that it will generate an exception right away if both command line args aren't included.